tests/serialize/structures.n3

```diff
@@ -0,0 +1,35 @@
+
+
+:MLK :name "Martin Luther King, Jr";
+  :born 1929-01-17; :died 1968-01-15;
+  :spouse [
+    :name "Coretta Scott King";
+    is :spouse of [
+      a :Marriage;
+      :spouse :MLK;
+      :offspring ( :MLK3 :DSK :YK [:name "Bernice King"]);
+    ]
+  ];
+
+  :wikipediapage <https://en.wikipedia.org/wiki/Martin_Luther_King_Jr.>.
+
+:MLK3 :name "Martin Luther King III".
+:DSK :name "Dexter Scott King".
+:YK :name "Yolanda King".
+
+:MLK :friend [].  # bnode with only one incoming arc
+[] :friend :MLK.  # bnode with only one outgoing arc
+
+[] :loves [].  # A floating arc between two bnodes
+
+:MLK is :author of  [a :Speech ; :date 1963-08-23; :title "I have a dream"] .
+
+# Datatypes
+:ZooReading1
+  :enabled true;
+  :disabled false;
+  :count 16;
```

Comment:
That's a hand-written input file, not an output file ... do not expect it to be ordered.